### ISSUE TITLE
Cannot download test symbols

### ISSUE THREAD

**Issue by chris-amco** [microsoft/navcontainerhelper#487]
I have just created a new container and have found that I am unable to download TEST symbols from it. I can download Application and System symbols. I included -includeTestToolkit when I created the container.

New-NavContainer command :
`New-NavContainer -containerName D365BC -accept_eula -imageName "bcinsider.azurecr.io/bcsandbox:th" -includeAL -includeCSide -doNotExportObjectsToText -useBestContainerOS -includeTestToolkit -WebClientPort 8081 -enableSymbolLoading -PublishPorts 7045,7046,7047,7048,7049,8081 -licenseFile "D:\Code\D365BC\Localise\AppSource\License\6010230.flf" -isolation hyperv`

(Host is Windows 10 1803 build 17134.765)

Results of Get-NAVAppInfo on container PS:
```
Get-NAVAppInfo -serverinstance NAV -symbolsonly
WARNING: This license is not compatible with this version of Business Central.
Id            : 486a1cbd-d9d0-443b-a6db-80bd7dd4165c
Name          : Application
Version       : 14.1.32615.0
Publisher     : Microsoft
ExtensionType : ModernDev
Scope         : Global

Id            : 8874ed3a-0643-4247-9ced-7a7002f7135d
Name          : System
Version       : 14.0.32600.0
Publisher     : Microsoft
ExtensionType : ModernDev
Scope         : Global
```

"Test" is missing.

Try to re-import the test toolkit: 
```
Import-TestToolkitToBCContainer -containername D365BC
Importing Objects from C:\TestToolKit\CALTestCodeunits.W1.fob (container path)
Importing Objects from C:\TestToolKit\CALTestLibraries.W1.fob (container path)
Importing Objects from C:\TestToolKit\CALTestRunner.fob (container path)
TestToolkit successfully imported
```

=> I confirmed that the test toolkit was present by successfully exporting a couple of its codeunits.
=> After this, Get-NAVAppInfo still does not show "Test".

Try regenerating symbols (container CMD):

```
finsql.exe Command=generatesymbolreference, Database="CRONUS", ServerName=localhost\SQLEXPRESS
```
Symbol regeneration was successful (navcommandresult.txt):
`[0] [06/11/19  5:20:57 AM] The command completed successfully in '529' seconds.`

=> After this, Get-NAVAppInfo still does not show "Test".

Have I missed something? Is there anything else I can try, apart from recreating the container?

Thanks in advance for any suggestions!
Chris

------------------------
NAV Server configuration (container PS)
```
Get-NAVServerConfiguration -serverinstance NAV

key                                             value
---                                             -----
AllowSessionWhileSyncAndDataUpgrade             false
AmountOfSqlStatementsInDebugger                 10
ApiServicesEnabled                              True
ApiSubscriptionDelayTime                        30000
ApiSubscriptionExpiration                       3
ApiSubscriptionMaxNumberOfNotifications         100
ApiSubscriptionMaxNumberOfSubscriptions         200
ApiSubscriptionNotificationUrlTimeout           5000
ApiSubscriptionsEnabled                         true
ApiSubscriptionSendingNotificationTimeout       30000
AppIdUri
AzureActiveDirectoryClientCertificateThumbprint
AzureActiveDirectoryClientId
AzureActiveDirectoryClientSecret
AzureKeyVaultClientCertificateStoreLocation     LocalMachine
AzureKeyVaultClientCertificateStoreName         My
AzureKeyVaultClientCertificateThumbprint
AzureKeyVaultClientId                           00000000-0000-0000-0000-000000000000
AzureKeyVaultKeyUri
BufferedInsertEnabled                           true
CalendarTwoDigitYearMax                         -1
ClientBuildRestriction                          WarnClient
ClientServicesAllowedFileTypes
ClientServicesChunkSize                         28
ClientServicesCompressionThreshold              64
ClientServicesCredentialType                    Windows
ClientServicesEnabled                           true
ClientServicesFederationMetadataLocation
ClientServicesIdleClientTimeout                 MaxValue
ClientServicesKeepAliveInterval                 00:02:00
ClientServicesMaxConcurrentConnections          500
ClientServicesMaxItemsInObjectGraph             512
ClientServicesMaxNumberOfOrphanedConnections    20
ClientServicesMaxUploadSize                     150
ClientServicesOperationTimeout                  MaxValue
ClientServicesPort                              7046
ClientServicesProhibitedFileTypes               ade;adp;asp;bas;bat;chm;cmd;com;cpl;csh;exe;fxp;gadget;hlp;hta;inf;ins;isp;its;js;jse;ksh;lnk;mad;maf;mag;mam;maq;mar;mas;mat;mau;mav;maw;mda;mdb...
ClientServicesProtectionLevel                   EncryptAndSign
ClientServicesReconnectPeriod                   00:10:00
ClientServicesTokenSigningKey
CompileBusinessApplicationAtStartup             true
DatabaseInstance                                SQLEXPRESS
DatabaseName                                    CRONUS
DatabaseServer                                  localhost
DataCacheSize                                   10
DebuggingAllowed                                true
DefaultClient                                   Web
DefaultLanguage
DeltaServiceClientTimeout                       00:02:00
DeveloperServicesEnabled                        true
DeveloperServicesPort                           7049
DeveloperServicesSSLEnabled                     false
DisableQueryHintForceOrder                      true
DisableQueryHintLoopJoin                        true
DisableQueryHintOptimizeForUnknown              false
DisableSmartSql                                 false
DisableTokenSigningCertificateValidation        false
DisableWriteInsideTryFunctions                  true
EnableALServerFileAccess                        true
EnableApplicationChannelLog                     true
EnableDataExportImport                          true
EnableDeadlockMonitoring                        true
EnableDebugging                                 false
EnableFullALFunctionTracing                     false
EnableLongRunningSqlStatementsInDebugger        true
EnableMembershipEntitlement
EnableSaveFromReportPreview                     true
EnableSaveToExcelForRdlcReports                 true
EnableSaveToWordForRdlcReports                  true
EnableSqlConnectionEncryption                   false
EnableSqlInformationDebugger                    true
EnableSymbolLoadingAtServerStartup              True
EnableTaskScheduler                             false
EncryptionProvider                              LocalKeyFile
ExcelAddInAzureActiveDirectoryClientId
ExchangeAuthenticationMetadataLocation
ExtendedSecurityTokenLifetime                   0
ExtensionAllowedTargetLevel                     Internal
ExternalTraceLevel                              Error
LockoutPolicyFailedAuthenticationCount          0
LockoutPolicyFailedAuthenticationWindow         0
LongRunningSqlStatementsInDebuggerThreshold     500
ManagementServicesEnabled                       true
ManagementServicesPort                          7045
MaxConcurrentCalls                              1000
MaxRowsToExportToExcel                          MaxValue
MaxStreamReadSize                               1000000
Multitenant                                     False
NASServicesEnableDebugging                      false
NASServicesRunWithAdminRights                   false
NASServicesStartupArgument
NASServicesStartupCodeunit
NASServicesStartupMethod
NavHttpClientMaxResponseContentSize             150
NavHttpClientMaxTimeout                         00:05:00
NetworkProtocol                                 Default
NonInteractiveSessionsLogRetainInterval         5.00:00:00
ODataEnableExcelAddInAnnotations                true
ODataMaxConnections                             0
ODataMaxConnectionsPerTenant                    0
ODataServicesEnabled                            true
ODataServicesMaxPageSize                        20000
ODataServicesOperationTimeout                   MaxValue
ODataServicesPort                               7048
ODataServicesSSLEnabled                         false
ODataServicesV3EndpointEnabled                  true
ODataServicesV4EndpointEnabled                  true
OverwriteExistingTranslations                   true
PublicODataBaseUrl                              http://D365BC:7048/NAV/OData
PublicSOAPBaseUrl                               http://D365BC:7047/NAV/WS
PublicWebBaseUrl                                http://D365BC:8081/NAV/
PublicWinBaseUrl                                DynamicsNAV://D365BC:7046/NAV/
ReplaceReportExecutionTimeWithClientTime        true
ReportAppDomainIsolation                        true
ReportPDFFontEmbedding                          true
SearchTimeout                                   00:00:10
SecurityProtocol                                Tls12
ServerInstance                                  NAV
ServicesCertificateThumbprint
ODataServicesOperationTimeout                   MaxValue
ODataServicesPort                               7048
ODataServicesSSLEnabled                         false
ODataServicesV3EndpointEnabled                  true
ODataServicesV4EndpointEnabled                  true
OverwriteExistingTranslations                   true
PublicODataBaseUrl                              http://D365BC:7048/NAV/OData
PublicSOAPBaseUrl                               http://D365BC:7047/NAV/WS
PublicWebBaseUrl                                http://D365BC:8081/NAV/
PublicWinBaseUrl                                DynamicsNAV://D365BC:7046/NAV/
ReplaceReportExecutionTimeWithClientTime        true
ReportAppDomainIsolation                        true
ReportPDFFontEmbedding                          true
SearchTimeout                                   00:00:10
SecurityProtocol                                Tls12
ServerInstance                                  NAV
ServicesCertificateThumbprint
ServicesCertificateValidationEnabled            true
ServicesDefaultCompany
ServicesDefaultTimeZone                         UTC
ServicesLanguage                                en-US
ServicesOptionFormat                            OptionCaption
ServicesUseNTLMAuthentication                   false
SessionEventTableRetainInterval                 90.00:00:00
SOAPServicesEnabled                             true
SOAPServicesMaxMsgSize                          1024
SOAPServicesPort                                7047
SOAPServicesSSLEnabled                          false
SqlBulkImportBatchSize                          448
SqlCommandTimeout                               00:30:00
SqlConnectionIdleTimeout                        00:05:00
SqlConnectionTimeout                            00:01:30
SqlLongRunningThreshold                         1000
SqlParametersByOrdinal                          true
SupportedLanguages
SqlConnectionIdleTimeout                        00:05:00
SqlConnectionTimeout                            00:01:30
SqlLongRunningThreshold                         1000
SqlParametersByOrdinal                          true
SupportedLanguages
TaskSchedulerMaximumConcurrentRunningTasks      10
TaskSchedulerSystemTaskEndTime                  23:59:59
TaskSchedulerSystemTaskStartTime                00:00:00
TokenSigningCertificateValidationMode           IssuerNameValidation
TraceLevel                                      Normal
UIElementRemovalOption                          LicenseFileAndUserPermissions
UseFindMinusWhenPopulatingPage                  true
UseIncrementalCompanyDelete                     true
UseSimplifiedFilters                            false
ValidAudiences
WSFederationLoginEndpoint
XmlMetadataCacheSize                            500
```

**Comment by chris-amco** [comment]
I tried re-creating the container with a simplified New-NavContainer command and I found that I was able to download the Test symbols.

To simplify the command, I removed the options -includeAL and -enableSymbolLoading:
`New-NavContainer -containerName D365BC -accept_eula -imageName "bcinsider.azurecr.io/bcsandbox:th" -includeCSide -doNotExportObjectsToText -useBestContainerOS -includeTestToolkit -WebClientPort 8081 -PublishPorts 7045,7046,7047,7048,7049,8081 -licenseFile "D:\Code\D365BC\Localise\AppSource\License\6010230.flf" -isolation hyperv`

I don't know which of these options was the cause of my issue, or whether it was caused by the combination of the two.
I had previously tried the original command but with the image "bcinsider.azurecr.io/bcsandbox" instead of "bcinsider.azurecr.io/bcsandbox:th" and found the same problem.

Regards,
Chris
